src/components/Person/Person.jsx

```diff
@@ -1 +1,28 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+      <p className="Person__partner">
+        {
+          !isMarried ? 'I am not married'
+            : `${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`
+        }
```

Comment:
CL#2 : Format ternary operator operands correctly - move each operand to the separate line
CL#3: Create separate variable for the condition.